Specs/Scene/ScreenSpaceCameraControllerSpec.js

```diff
@@ -507,7 +507,6 @@ defineSuite([
         moveMouse(MouseButtons.LEFT, startPosition, endPosition, true);
         updateController();
         expect(camera.position).toEqual(position);
-        expect(camera.direction).not.toEqual(Cartesian3.normalize(Cartesian3.negate(camera.position, new Cartesian3()), new Cartesian3()));
```

Comment:
I removed this line because it wasn't actually doing anything. It only passed before because (0,0,-0.999999) did not equal (0,0,-1.0). Or maybe it should be checking `.toEqual` instead of `.not.toEqual`?